index.html

```diff
@@ -47,11 +48,13 @@
           </ul>
         </div>
 
-        <button>Donate Now</button>
+        <button class="btn">Donate Now</button>
+
+      </header>
 
-    </div>
 
-    <div class="content">
+    <!--main start-->
+    <main class="content">
         <div class="main">
```

Comment:
Best to name the 2nd div `main` and leave the first one as a div. That way, your `aside` further down is not nested inside the `main`.
It's ok for `main` and `aside` to be wrapped inside that `div` with the 'content' class - it might be there for the purposes of CSS. Have you checked what the `.content` class does in the CSS?

Reply:
I changed the 2nd div to main and the site behaved weirdly. I think I'll go back and have a rethink and change this.